commands/command_prune.go

```diff
@@ -176,28 +177,29 @@ func prune(fetchPruneConfig lfs.FetchPruneConfig, verifyRemote, dryRun, verbose
 	}
 
 	if len(prunableObjects) == 0 {
-		Print("Nothing to prune")
 		return
 	}
+
+	info := tasklog.NewSimpleTask()
+	logger.Enqueue(info)
 	if dryRun {
-		Print("%d files would be pruned (%v)", len(prunableObjects), humanize.FormatBytes(uint64(totalSize)))
+		info.Logf("prune: %d file(s) would be pruned (%s)", len(prunableObjects), humanize.FormatBytes(uint64(totalSize)))
 		if verbose {
-			Print(verboseOutput.String())
+			for _, item := range verboseOutput {
```

Comment:
I think you can take out the two extra `if verbose` (here and L194), since `verboseOutput` will be empty in that case.

Reply:
Good, call -- I changed this in 782c58c3.